### ISSUE TITLE
We need a new contributor

### ISSUE THREAD

**Issue by astrism** [formly-js/react-formly#10]
I don't currently have time to work on the project, and I think @kentcdodds is on to other things as well, anyone want to take over?

**Comment by kentcdodds** [comment]
Yup, also not using this now. I'll hand the keys over to anyone who currently uses this library.
